console/console-init/ui/src/modules/device-detail/components/DeviceInfoGateways/DeviceInfoGateways.tsx

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2020, EnMasse authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+import React from "react";
+import {
+  Grid,
+  GridItem,
+  Card,
+  CardBody,
+  CardHeader,
+  Title,
+  Button
+} from "@patternfly/react-core";
+
+interface DeviceInfo {
+  id: string;
+  name: string;
+}
+
+export interface DeviceInfoGatewaysProps {
```

Comment:
interface name should start with I as per convention.
Like IDeciveInfoGatewayProps